### ISSUE TITLE
Skill Check Crashes On Launch when less than "49" Mods are loaded...

### ISSUE THREAD

**Issue by lizfinite** [CottonMC/SkillCheck#19]
### I know. 
**"What the $%@#? How? How did you even test this? Why are you making issues on all of my mods!?"**

### To answer those questions:
I'm not sure why it's happening, but I was able to reproduce it consistently with slightly different sets of mods.

I was originally testing my Mod Pack for a different issue that ended up being caused by Skill Check. 
When I went to verify this issue in a fresh instance with only Skill Check, Cotton and Fabric API, I was getting a [crash](https://pastebin.com/DDbAEZt2). 

_"That's odd..."_ I thought. *"I'm able to run my 90+ mods mod pack without crashing... and that has Skill Check?"* 

My first thought was that another mod was bundling an older or required dependency of skill check. So I created a fresh instance of my mod pack and started removing mods in batches, skipping dependencies.
When I recreated the [crash](https://pastebin.com/J0vAAeY1), I readded the last group; then removed mods from that group until it happened again. **Huzzah!** [We found the mod keeping Skill Check afloat!](https://pastebin.com/wphih4XF)
So I thought... But... it gets weirder... 

I then added that mod to the fresh instance thinking I could begin to reproduce the other non-crashing issue; and we're still crashing on launch.
*"What's going on here?"* 

By now I'm confused. So I create a second instance and begin removing mods in inverse order from before. After some time the crash occurs again... It was at this time I noticed that the "Mod Menu" had been showing 49 (or more) mods loaded whenever I successfully got to the menu. 
So, to confirm my suspicions I created a [3rd instance](https://pastebin.com/Fq8Zfbz6) and randomly selected mods from my pack to add to it. After adding enough to not crash, I removed mods again, one by one. The "mods loaded" section counting down to 49, until upon the next launch I [crashed.](https://pastebin.com/Rbpvuu6X) 
I then readded the last removed mod and removed a different one in that same instance. [Crashed.](https://pastebin.com/AKL4XniP) This confirmed my suspicions.

Also I'm creating issues because I use all your mods with my friends in my ever updating mod pack and we ❤️ them, so I hope I'm helping!
Ironically this doesn't affect me, but the issue that does (right clicking with any items attacks entities when crouching (rebound to L-Ctrl)) can't be tested without having other mods installed

### **TL;DR** Game crashing seemingly because there's less than 49* mods with a line related to `could not find any targets matching 'disconnect'`
Logs below.

**This isn't actually* **49** *mods in each log, I only say such in the title because that's the amount of mods that show in "Mod Menu" mod when I don't crash and get to the menu, due to Fabric API 0.3.0's modularity showing up as multiple mods.*

- [Alphabetical mods no crash](https://pastebin.com/wphih4XF)
- [Alphabetical mods instance crash 1](https://pastebin.com/J0vAAeY1)
- [Alphabetical mods instance crash 2](https://pastebin.com/a2riLa0r)
- [Random mods instance no crash](https://pastebin.com/Fq8Zfbz6)
- [Random mods instance crash 1](https://pastebin.com/Rbpvuu6X)
- [Random mods instance crash 2](https://pastebin.com/AKL4XniP) 
- [Another instance no crash](https://pastebin.com/MWqhd34N)
- [Another instance Crash](https://pastebin.com/iDqWM6ez)
- [No Mods Crash](https://pastebin.com/DDbAEZt2)

**Comment by LemmaEOF** [comment]
That is *truly* bizarre. I can only hope that it's some mappings issue with nested Fabric API, so try [this jar](https://dsc.cloud/b0undarybreaker/skillcheck-1.0.9-1.14.2-test.jar) in a crashing instance and see if that changes anything?

**Comment by lizfinite** [comment]
Just tried the Test jar in an instance with only the latest cotton and Fabric API 0.7.3, still getting a crash on launch related to the same error: `could not find any targets matching 'disconnect'`

- https://pastebin.com/kiYBzmnD (No other mods)

As well, I tried it in an instance with enough mods not to crash and then relaunched after having removed a mod:
- https://pastebin.com/9PkyE8rW (No Crash with other mods)
- https://pastebin.com/pSMpDu2B (Crash with mods, removed Redstone bits in this one)
- https://pastebin.com/65VDBBk7 (Crash with mods 2, removed Nbtcrafting in this one)

So strange...

**Comment by sargunv** [comment]
Not sure what's up with the 49 mod thing yet, but this appears to be an issue in a recent version of fabric-networking-v0. This mod launches just fine with itself, its included mods, and modmenu, but as soon as I add fabric-networking-v0-0.1.2+200eb5c2, the crash happens. Will dig further.

**Comment by sargunv** [comment]
Managed to reproduce this with just fabric API and Crochet, so it's definitely not a SkillCheck issue, just an issue with either Crochet or Fabric API.

~~EDIT: wait, nvm, can't reproduce with Crochet anymore. I got mixed up somewhere.~~

**Comment by sargunv** [comment]
Ah, got it. Crochet bundles fabric-networking, which I removed the second time I tried it. Crochet's bundled fabric-networking + the latest fabric-networking-v0 installed together reproduce the crash.

**Comment by LemmaEOF** [comment]
I'm gonna close this here since I'm probably moving away from Crochet now that it's unmaintained
